docs/cookbook/admin_panel/index.md

```diff
@@ -1,208 +1,8 @@
-Choose the YAML or the PHP version.
-
-```yaml
-# config/packages/sylius_bootstrap_admin_ui.yaml
-# ...
-sylius_twig_hooks:
-    hooks:
-        # ...
-        'sylius_admin.common.component.sidebar.logo':
-            image:
-                # template: '@SyliusBootstrapAdminUi/shared/crud/common/sidebar/logo/image.html.twig'
-                template: 'shared/crud/common/sidebar/logo/image.html.twig'
-
-```
-
-```php
-// config/packages/sylius_bootstrap_admin_ui.php
-use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
-
-return static function (ContainerConfigurator $containerConfigurator): void {
-    // ...
-
-    // Add these following lines to define your own Twig template for the logo.
-    $containerConfigurator->extension('sylius_twig_hooks', [
-        'hooks' => [
-            'sylius_admin.common.component.sidebar.logo' => [
-                'image' => [
-                    // 'template' => '@SyliusBootstrapAdminUi/shared/crud/common/sidebar/logo/image.html.twig',
-                    'template' => 'shared/crud/common/sidebar/logo/image.html.twig',
-                ],
-            ],
-        ],
-        
-    ]);
-};
-
-```
-
-```twig
-{% raw %}
-{# templates/shared/crud/common/sidebar/logo/image.html.twig #}
-
-<img src="{{ asset('images/logo.png') }}" alt="Your Brand name" class="navbar-brand-image" />
-{% endraw %}
-  
-```
-
-## How to customize the login page logo
-
-To customize the login page logo,you need to set new logo template at `sylius_admin.security.login.logo` twig hook.
-Choose the YAML or the PHP version.
-
-```yaml
-# config/packages/sylius_bootstrap_admin_ui.yaml
-# ...
-sylius_twig_hooks:
-    hooks:
-        # ...
-        'sylius_admin.security.login.logo':
-            image:
-                # template: '@SyliusBootstrapAdminUi/security/common/logo/image.html.twig'
-                template: 'security/common/logo/image.html.twig'
-
-```
-
-```php
-// config/packages/sylius_bootstrap_admin_ui.php
-use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
-
-return static function (ContainerConfigurator $containerConfigurator): void {
-    // ...
-
-    // Add these following lines to define your own Twig template for the logo.
-    $containerConfigurator->extension('sylius_twig_hooks', [
-        'hooks' => [
-            'sylius_admin.security.login.logo' => [
-                'image' => [
-                    // 'template' => '@SyliusBootstrapAdminUi/security/common/logo/image.html.twig'
-                    'template' => 'security/common/logo/image.html.twig',
-                ],
-            ],
-        ],
-    ]);
-};
-```
-
-```twig
-{% raw %}
-<img src="{{ asset('images/logo.png') }}" alt="Your Brand name" class="sylius navbar-brand-image">
-{% endraw %}
-```
+* [Basic operations](basic_operations.md)
+* [Customizing the_logo](logo.md)
+* [Customizing the menu](menu.md)
 
 ## How to customize the admin menu
```

Comment:
This section about admin menu is extracted to separate file as well, hence it needs to be removed from the index.md

```suggestion
```

Reply:
done, indeed, I forgot to remove this part. thx.